### DISCUSSION TITLE
How to highlight gems to buy

### DISCUSSION THREAD

**Discussion by velit** [Lailloken/Exile-UI#419] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/82089795-a33b-4009-b9c6-8068e488bab9)

When I press the omni key (middle click) and hold it, it first closes the vendor window and after the hold period opens my passive tree. What's the intended way to invoke the behavior to highlight the gems?

If I press the omni key on a gem it just shows me "No notes for this gem". Is the intended way to iterate all the gems and check the note?

**Comment by velit** [comment, marked as answer]
Ah figured it out. You're supposed to press and hold omni key on the Highlight items input box.

**Reply by Lailloken** [reply]
Anywhere within the vendor window should be fine. I'm curious why it behaved like that initially, though. Do you use custom in-game binds?

**Reply by velit** [reply]
I very much do, although I tried to check all the inputs that are in Hotkeys settings if they differed from default

**Reply by velit** [reply]
![image](https://github.com/user-attachments/assets/8ddba85e-3b55-49ef-b953-0d9d20c01144)
![image](https://github.com/user-attachments/assets/996f9793-c47c-46cd-a571-8523a1a12481)
![image](https://github.com/user-attachments/assets/d2bb76c1-9da1-4bb9-9ba5-232d588b89aa)
![image](https://github.com/user-attachments/assets/6decb500-8786-49ae-b622-947aae0654a9)
![image](https://github.com/user-attachments/assets/3bfb52e8-2edb-450e-aa6e-07bc83d5b1f0)
![image](https://github.com/user-attachments/assets/4c0b1021-a89a-4d5c-a51a-d638fbac3006)
![image](https://github.com/user-attachments/assets/01cb89a4-48ce-4a99-9a98-fe52fd394efb)
![image](https://github.com/user-attachments/assets/6a167fab-8640-4294-bbd0-bc7f91724204)
![image](https://github.com/user-attachments/assets/bbcf285f-aa8e-486d-ad74-9d492541ff28)

If there's anything that would help

**Reply by Lailloken** [reply]
Thanks for being so thorough with this. I tried to replicate it by copying certain keybinds, but I can't reproduce this behavior.
When long-pressing the omni-key while the gem-prompt is on screen, the tool sends CTRL-F, then CTRL-V-Enter.
It's also weird that it waited for the omni-key to be released and then opened your skill tree.

Well, if it works while hovering the search-bar, then it's fine for now. GLHF

**Reply by velit** [reply]
Just realized I have C bound to something else than character screen. I've been playing other games where the character screen is essentially an inventory that I forgot poe has a separate character screen. I'll try with the correct config.

**Reply by velit** [reply]
The middle click does work properly when the I had corrected the config options 
![image](https://github.com/user-attachments/assets/9600ffc8-7673-490e-88f3-9f4ac9079bca)

**Reply by Lailloken** [reply]
Oh, I missed that on your screenshots. Yeah, it's unfortunately a weird behavior on the game's side. When you CTRL-C but don't actually copy an item's info, nothing happens -- UNLESS you have rebound the c-key to something other than the character screen. The omni-key uses CTRL-C to check if you're trying to activate an item-related feature or anything else.

**Reply by velit** [reply]
Hmm, I bound a useless action to Ctrl-C which prevents the game to default to the C action but it still works for copying items. Specifically I bound use apex sentinel to Ctrl-C, I'll test if this allows me to still use just one omni key. At least cursory testing indicates this works but I'll give info if I run into issues during the league start.
